cassandra/src/main/scala/filodb.cassandra/FiloCassandraConnector.scala

```diff
@@ -63,7 +63,7 @@ trait FiloCassandraConnector extends StrictLogging {
                  _: ConnectionException ) =>
         logger.warn(s"CQL execution of $statement resulted in error. Will retry $retryAttemptsLeft times", e)
         if (retryAttemptsLeft > 0) {
-          FiloCassandraConnector.cassRetriesScheduledCount.increment()
+          FiloCassandraConnector.cassRetriesScheduledCount.increment
```

Comment:
Shouldn't it be `increment()` ? usually `()` is associated with method that has side effect which this one has.

If they truly changed the API, it would be a confusing change. Cant tell whether increment operation is being done or `increment` attribute is being accessed. Quite bad!

Reply:
Ok. This is taken care of now.